src/content/getusermedia/getdisplaymedia/js/main.js

```diff
@@ -40,7 +47,12 @@ function errorMsg(msg, error) {
 
 const startButton = document.getElementById('startButton');
 startButton.addEventListener('click', () => {
-  navigator.mediaDevices.getDisplayMedia({video: true, audio: true})
+  const options = {audio: true, video: {}};
+  const displaySurface = preferredDisplaySurface.options[preferredDisplaySurface.selectedIndex];
+  if (displaySurface.value !== '') {
```

Comment:
nit: Is this the idiomatic thing to do? What about `if (displaySurface.value) {`?

Reply:
idiomatic yes, good for a sample no

Reply:
changed to `!= 'default'` which is easier to understand